README.rst

```diff
@@ -541,6 +554,11 @@ Choose from the list of available fixers:
                         the class name should match
                         the file name. (Risky fixer!)
 
+* **random_api_migration**
```

Comment:
Added configuration method: https://github.com/kalessil/PHP-CS-Fixer/blob/features_dirconst_randapi_typecast/Symfony/CS/Fixer/Contrib/RandomApiMigrationFixer.php#L37

@keradus : configure method already defined in a parent, so making it static not possible